COMPATIBILITY.md

```diff
@@ -5,23 +5,12 @@
 
 The below matrix shows the compatibility of the [`opensearch-project/opensearch-php`](https://packagist.org/packages/opensearch-project/opensearch-php) with versions of [`OpenSearch`](https://opensearch.org/downloads.html#opensearch).
 
-| OpenSearch Version | Client Version |
-| --- | --- |
-| 1.0.0 | 1.0.0 |
-| 1.0.1 | 1.0.0 |
-| 1.1.0 | 1.0.0 |
-| 1.2.0 | 1.0.0 |
-| 1.2.1 | 1.0.0 |
-| 1.2.2 | 1.0.0 |
-| 1.2.3 | 1.0.0 |
-| 1.2.4 | 1.0.0 |
-| 1.3.0 | 1.0.0 |
-| 1.3.1 | 1.0.0 |
-| 1.3.2 | 1.0.0 |
-| 1.3.3 | 1.0.0 |
-| 2.0.0 | 2.0.0 |
-| 2.0.1 | 2.0.0 |
 
-## Upgrading
+|  | OpenSearch 1.0 | OpenSearch 2.0 |
```

Comment:
How about if we simplify it like https://github.com/opensearch-project/opensearch-java/pull/183/files#diff-a87f2d0c41bfc8695da75503ee3986471c4a0c6fa642e066745302bf88dcce23L8?

Reply:
@wjonkerhulst do you want to do this? or should I take it with me?

Reply:
@shyim You can take it. This was just a suggestion. The example from @VachaShah is also fine with me.